kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ContainerInstanceDescriptor.java

```diff
@@ -33,8 +34,7 @@ public class ContainerInstanceDescriptor {
     private String containerImage;
     private String containerImageTag;
     private String containerID;
-    private List<Integer> containerPortsExternal;
-    private List<Integer> containerPortsInternal;
+    private List<ContainerPort> containerPorts = new ArrayList<>();
```

Comment:
Should we add a getter for this field that returns a `List<ContainerPort>`? Otherwise the protocol will not be visible to clients.